src/test/java/com/google/sps/servlets/UserDataServletTest.java

```diff
@@ -139,24 +137,23 @@ public void testGetMethodWithValidId() throws Exception {
     writer.flush();
 
     // Build the correct response output
-    Map<String, Object> correctResponse = new HashMap<>();
-    correctResponse.put(USER_FOUND_PROPERTY, true);
-    correctResponse.put(USER_NAME_PROPERTY, TEST_USER_NAME);
-    correctResponse.put(USER_ID_PROPERTY, TEST_USER_ID);
-    correctResponse.put(USER_EMAIL_PROPERTY, TEST_USER_EMAIL);
-    correctResponse.put(USER_BIO_PROPERTY, TEST_USER_BIO);
-    correctResponse.put(USER_FRIENDS_LIST_PROPERTY, Arrays.asList(TEST_USER_FRIENDS_LIST));
+    ImmutableMap.Builder<String, Object> expected = ImmutableMap.builder();
```

Comment:
You can simplify this just a tad:

```
ImmutableMap<String, Object> expected = ImmutableMap.of(
  USER_FOUND_PROPERTY, true,
  USER_BIO_PROPERTY,  TEST_USER_BIO,
  ...
);
```

Reply:
https://github.com/googleinterns/step48-2020/pull/13#discussion_r452506036. Mentioned this in one of the comments from the previous review, but I resolved it.

Reply:
Let me know if you think using `of` is still possible!

Reply:
Sorry, I can't find that. Why is `of()` a problem here?

Reply:
`of()` takes up to 5 key-value pairs in its argument. Here is a link to the `of()` documentation: [of()](https://guava.dev/releases/23.0/api/docs/com/google/common/collect/ImmutableMap.html#of--)